client/packages/react/src/InstantReactAbstractDatabase.tsx

```diff
@@ -197,10 +204,8 @@ export default abstract class InstantReactAbstractDatabase<
   useQuery = <Q extends ValidQuery<Q, Schema>>(
     query: null | Q,
     opts?: InstaQLOptions,
-  ): InstaQLLifecycleState<
-    Schema,
-    Q,
-    NonNullable<Config['useDateObjects']>
+  ): DataAlwaysDefined<
```

Comment:
Should we update InstaQLLifecycleState itself? (I am not sure why DataAlwaysDefined is necessary)

Reply:
we'd have to duplicate the whole type because the non-react packages also use the type